plone/app/z3cform/tests/test_widgets.py

```diff
@@ -201,20 +217,22 @@ def setUp(self):
         self.field = Datetime(__name__='datetimefield')
         self.widget = DatetimeWidget(self.request)
         self.widget.pattern_options = {
-            'date': {'firstDay': 0},
             'time': {'interval': 15}
         }
 
+    @mock.patch('plone.app.z3cform.widget.first_weekday', new=lambda: 3)
     def test_widget(self):
         current_year = datetime.today().year
+        import ipdb
+        ipdb.set_trace()
```

Comment:
Ooops! :)
That's the dev version which wasn't supposed to be checked in. It even has some changes to other tests, which are not part of this PR. Tnx for the pointer!